scripts.py

```diff
@@ -217,6 +217,11 @@ def main_logic(idx):
                         link = "http://" + link
                     print("[scripts] " + coords + "    Open website " + link + " in default browser, try to make a new window")
                     webbrowser.open_new(link)
+                elif split_line[0] == "CODE":
+                    args = " ".join(split_line[1:])
+                    print("[scripts] " + coords + "    Opening " + args)
+                    print(args)
```

Comment:
This second print statement can be removed

Reply:
true i noticed this morning that if forgot to remove the second print